packages/stryker/test/unit/reporters/ClearTextReporterSpec.ts

```diff
@@ -33,22 +37,60 @@ describe('ClearTextReporter', function () {
       });
 
       it('should report on the survived mutant', () => {
-        expect(process.stdout.write).to.have.been.calledWith('Mutator: Math\n');
-        expect(process.stdout.write).to.have.been.calledWith(chalk.red('-   original line') + '\n');
-        expect(process.stdout.write).to.have.been.calledWith(chalk.green('+   mutated line') + '\n');
+        expect(process.stdout.write).to.have.been.calledWithMatch(sinon.match('Mutator: Math'));
+        expect(process.stdout.write).to.have.been.calledWith(chalk.red('-   original line') + os.EOL);
+        expect(process.stdout.write).to.have.been.calledWith(chalk.green('+   mutated line') + os.EOL);
       });
 
       it('should not log individual ran tests', () => {
-        expect(process.stdout.write).to.not.have.been.calledWith('Tests ran: \n');
-        expect(process.stdout.write).to.have.been.calledWith('Ran all tests for this mutant.\n');
+        expect(process.stdout.write).to.not.have.been.calledWithMatch(sinon.match('Tests ran:'));
+        expect(process.stdout.write).to.have.been.calledWithMatch(sinon.match('Ran all tests for this mutant.'));
+      });
+    });
+
+    describe('onScore', () => {
+      it('should report the clear text table with correct values', () => {
+        sut.onScore(scoreResult({
+          name: 'root',
+          killed: 1,
+          timedOut: 2,
+          survived: 3,
+          noCoverage: 4,
+          errors: 5,
+          mutationScore: 80,
+          childResults: [scoreResult({
+            name: 'child1',
+            mutationScore: 60,
+            childResults: [
+              scoreResult({
+                name: 'some/test/for/a/deep/file.js',
+                mutationScore: 59.99
```

Comment:
That was the problem. They sometimes were and sometimes were not. Now they're always kept raw.